### PR TITLE
docs: refine-kbar

### PR DESCRIPTION
**Pull request by biskuvit**
added [refine-kbar](https://github.com/pankod/refine/tree/master/packages/kbar) command palette document. 

### Self Check before Merge

Please check all items below before review.

-   [x] Corresponding issues are created/updated or not needed
-   [x] Docs are updated/provided or not needed
-   [x] Examples are updated/provided or not needed
-   [x] TypeScript definitions are updated/provided or not needed
-   [x] Tests are updated/provided or not needed
-   [x] Changesets are provided or not needed

### CODE REVIEW

File: documentation/docs/packages/command-palette/kbar.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+---
+id: kbar
+title: kbar
+---
+
+import example from '@site/static/img/packages/command-palette/kbar/refine-kbar-example.gif';
+
+**refine** support command palette feature and use the
```
Comment (salihozdemir) [inline documentation/docs/packages/command-palette/kbar.md:8]:
```suggestion
**refine** supports command palette feature and use the
```

File: documentation/docs/packages/command-palette/kbar.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+:::info
+If your are going to use the [`superplate`][superplate-url] to create new project, you can select the `kbar` as a command palette.
+
+:::
+
+## Basic Usage
+
+First of all, you need to import the `@pankod/refine-kbar` library and we will use `RefineKbarProvider` to wrap the whole application.
+After that, we should create the `<OffLayoutArea/>` component for the Refine component and use the `refine-kbar` command palette in `<OffLayoutArea>`. We have the `<RefineKbar>` component to provide the command palette to the `<Refine>` component.
```
Comment (salihozdemir) [inline documentation/docs/packages/command-palette/kbar.md:27]:
```suggestion
First of all, you need to import the `@pankod/refine-kbar` library and we will use `RefineKbarProvider` to wrap the whole application.

After that, we should create the `<OffLayoutArea/>` component for the `<Refine>` component and use the `refine-kbar` command palette in `<OffLayoutArea>`. We have the `<RefineKbar>` component to provide the command palette to the `<Refine>` component.
```

File: documentation/docs/packages/command-palette/kbar.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+</div>
+
+<br/>
+
+:::note
+_Why we need to add `<OffLayoutArea>` to the `<Refine>` component?_<br/>
+Because we need to reach the `resources` property of the `<Refine>` component.
+:::
+
+## Accessibility
```
Comment (salihozdemir) [inline documentation/docs/packages/command-palette/kbar.md:93]:
```suggestion
## Permissions
```

`useCanWithoutCache` is not relating to accessibility. Also I'm not sure delete is relating to accessibility. I've suggested code above but maybe we should think how to rename.

File: documentation/docs/packages/command-palette/kbar.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+For more information check out the source code of [`refine-kbar`][kbar] package
+
+## Custom Actions
+
+If we want to add custom actions to the command palette, we can use the `createAction` and `useRegisterActions` of the `refine-kbar`.
+check the [`refine-finefoods`][refine-finefoods] example. Also you can find more information about actions in the [`kbar`][kbar-actions] package.
+
+You can use the `createAction` to create a new action and use the `useRegisterActions` to register the action to the command palette.
+
+```tsx title="custom actions example"
```
Comment (salihozdemir) [inline documentation/docs/packages/command-palette/kbar.md:105]:
```suggestion
```tsx title="src/components/offLayoutArea.tsx"
```

I think, we should add this in example and we refer to example in this title.

File: documentation/docs/packages/command-palette/kbar.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+        <div class="control orange"></div>
+        <div class="control green"></div>
+    </div>
+    <img src={example} alt="Refine Kbar Example" />
+</div>
+
+<br/>
+
+:::note
+_Why we need to add `<OffLayoutArea>` to the `<Refine>` component?_<br/>
```
Comment (ozkalai) [inline documentation/docs/packages/command-palette/kbar.md:89]:
```suggestion
_Why do we need to add `<OffLayoutArea>` to the `<Refine>` component?_<br/>
```

File: documentation/docs/packages/command-palette/kbar.md (hunk 4 above)
Comment (biskuvit) [reply documentation/docs/packages/command-palette/kbar.md:105]: We have talked before with @aliemir and decided to keep the base example simple and had a custom example in our **fine foods** example. Which has mentioned in the documentation.

**Review by aliemir — APPROVED** [review]

File: documentation/docs/packages/command-palette/kbar.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+You can use the `createAction` to create a new action and use the `useRegisterActions` to register the action to the command palette.
+
+```tsx title="Custom action example"
+import { createAction, useRegisterActions } from "@pankod/refine-kbar";
+
+const customAction = createAction({
+    name: "my custom action",
+    section: "custom-actions",
+    perform: () => {
+        console.log("onClick my custom action");
```
Comment (omeraplak) [inline documentation/docs/packages/command-palette/kbar.md:117]:
```suggestion
        console.log("onSelect my custom action");
```

File: documentation/redirects.json [hunk 7]
```diff
@@ -1035,6 +1035,10 @@
         {
             "from": "/docs/ui-frameworks/antd/hooks/resource/useMenu/",
             "to": "/docs/core/hooks/ui/useMenu/"
+        },
+        {
+            "from": "/docs/packages/command-palette/kbar/",
+            "to": "/docs/packages/command-palette/"
```
Comment (omeraplak) [inline documentation/redirects.json:1041]:
```suggestion
            "from": "/docs/examples/command-palette/kbar/",
            "to": "/docs/examples/command-palette/"
```

**Review by omeraplak — CHANGES_REQUESTED** [review]

File: documentation/docs/getting-started/overview.md [hunk 8]
```diff
@@ -121,6 +120,9 @@ Follow the _CLI wizard_ to select options and start creating your project.
 ? Do you want a customized layout?:
 ❯ No
 
+? Do you want to add Kbar command pallete?:
```
Comment (omeraplak) [inline documentation/docs/getting-started/overview.md:123]: can you update other tutorials too?

**Review by omeraplak — CHANGES_REQUESTED** [review]

**Review by omeraplak — APPROVED** [review]

**Pull request merged**